src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ValidateVariants.java

```diff
@@ -203,6 +241,12 @@ public void apply(final VariantContext vc, final ReadsContext readsContext, fina
      * @return the final set of type to validate. May be empty.
      */
     private Collection<ValidationType> calculateValidationTypesToApply(final List<ValidationType> excludeTypes) {
+        if (VALIDATE_GVCF && !excludeTypes.contains(ValidationType.ALLELES)) {
```

Comment:
Should this be an error instead of a warning?  Why is GVCF validation implemented with it's own argument instead of being part of validation enum? (just curious, don't have to change it necessarily...)